presto-spi/pom.xml

```diff
@@ -13,6 +13,7 @@
 
     <properties>
         <air.main.basedir>${project.parent.basedir}</air.main.basedir>
+        <air.check.skip-dependency>true</air.check.skip-dependency>
```

Comment:
it's OK since it's removed in the next commit

Reply:
Yeah, this is kind of lame, but it's needed so this commit builds by itself, and I didn't want to mix in all the POM updates with code changes.